consent/handler.go

```diff
@@ -745,6 +745,7 @@ func (h *Handler) acceptOAuth2ConsentRequest(w http.ResponseWriter, r *http.Requ
 
 	var p flow.AcceptOAuth2ConsentRequest
 	d := json.NewDecoder(r.Body)
+	d.UseNumber()
```

Comment:
I don't think other "endpoints" has problems, the issue is specifically with the data saved into session. I didn't notice other places where this would be an issue.

Reply:
Hey, I added small helper and replaced occurrences of decode in several handlers. I'm not sure how can I use MarshalJSON methods with this change as there is only call to json.Marshal there. I'm also not really a go developer, so I'm not used to work with this language.